website/docs/d/tables.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mysql"
+page_title: "MySQL: mysql_tables"
+sidebar_current: "docs-mysql-datasource-tables"
+description: |-
+  Get tables on a MySQL server.
```

Comment:
Just nit-picking: please add "s" as it's 3rd person singular. So it `gets` tables from a MySQL server.

The same issue is with description a bit lower